src/dbnode/generated/thrift/rpc.thrift

```diff
@@ -490,6 +492,20 @@ struct Query {
   7: optional FieldQuery       field
 }
 
+struct AggregateTilesRequest {
+	1: required string sourceNameSpace
+	2: required string targetNameSpace
+	3: required i64 rangeStart
+	4: required i64 rangeEnd
+	5: required string step
+	6: bool removeResets
```

Comment:
Is this actually used yet? If not we should remove it until we need it, we don't really know the best way to specify this (is it part of the job definition, etc or a high level attribute like this here).

Reply:
It is used already, in one of the later branches. The idea is to let us experiment with the special counter downsampling logics before we have the type metadata on each series. After that, sure, this will be removed. Will add a FIXME for it.